### DISCUSSION TITLE
Postgresql Piece Implementation

### DISCUSSION THREAD

**Discussion by ghost** [anacrolix/torrent#494] [Show and tell]
Feel free to use https://github.com/anacrolix/torrent/blob/00d56381b580ee0433899f9702a592fbf4839880/storage/postgresql-piece-completion.go if you want to use Postgresql for piece completion

**Comment by anacrolix** [comment]
Thanks for the contribution.

**Reply by ghost** [reply]
Welcome. Thank you too for providing such good torrent library
